flask_server.py

```diff
@@ -93,20 +103,23 @@ def login_page():
         return render_template("login.html")
     elif request.method == "POST":
         if login_and_store_session_id(request.form["username"], request.form["password"]):
+            logger.info("User {} has logged in".format(request.form["username"]))
```

Comment:
As above,

POST/GET request to /login would be nicer (If post also log the username)